wgengine/router/router_windows.go

```diff
@@ -111,6 +111,13 @@ func (r *winRouter) Set(cfg *Config) error {
 		return fmt.Errorf("dns set: %w", err)
 	}
 
+	err = exec.Command("ipconfig", "/flushdns").Run()
```

Comment:
Does ifconfig program output useful error message when it fails? If it does, you can grab that message by using CombinedOutput method instead of Run. And then you can print that output into the log.

Reply:
To be honest, I don't know if `ipconfig /flushdns` prints any useful message on error because I've never encountered an error. But sure, it won't hurt to add the output.